app/views/layouts/_navbar.html.haml

```diff
@@ -6,18 +6,18 @@
       .navbar-brand= link_to "dot.paper", '/'
     .collapse.navbar-collapse#bs-navbar-collapse-1
       %ul.nav.navbar-nav.navbar-right
-        %li
-          = link_to "Shop", products_path
-        %li
-          = link_to "Cart", checkout_index_path
         -if current_user&.admin?
           %li
             = link_to "Admin Dashboard", admin_products_path
+        %li
+          = link_to "Shop", products_path
         - if user_signed_in?
           %li
+            = link_to "({{ order.order_items.length }})", checkout_index_path, class: "glyphicon glyphicon-shopping-cart"
```

Comment:
instead of calling this ```$scope.order```could call it ```$scope.cartOrder``` and then you'd be able to change all the ```$scope.ord``` back to ```$scope.order```